features/02_configure_aruba/activate_announcer_on_command_failure.feature

```diff
@@ -10,6 +10,8 @@ Feature: Configure announcer activation on command failure
   Scenario: Default value
     Given a file named "features/support/aruba.rb" with:
```

Comment:
Let's rename the file to say `aruba_config.rb` instead of adding `require ...`. This is not the thing we want to show the user in this example. The require is done in the fixture in `aruba.rb`.